backend/hitas/models/apartment.py

```diff
@@ -241,10 +241,13 @@ def sell_by_date(self):
                 if sell_by_date is None:
```

Comment:
You could refactor lines 240-256 to use the new `hitas.models.condition_of_sale.ConditionOfSale.sell_by_date` you added.

Reply:
Of course, looks like I completely forgot to do that after initially implementing it